composer.json

```
@@ -1,63 +1,75 @@
+	"license": "MIT",
+	"type": "phpcodesniffer-standard",
 	"keywords": [
 		"phpcs",
 		"static analysis",
 		"standards",
 		"WordPress"
 	],
-	"license": "MIT",
 	"authors": [
 		{
 			"name": "Contributors",
 			"homepage": "https://github.com/Automattic/VIP-Coding-Standards/graphs/contributors"
 		}
 	],
+	"support": {
+		"issues": "https://github.com/Automattic/VIP-Coding-Standards/issues",
+		"wiki": "https://github.com/Automattic/VIP-Coding-Standards/wiki",
+		"source": "https://github.com/Automattic/VIP-Coding-Standards"
+	},
 	"require": {
 		"php": ">=5.4",
 		"dealerdirect/phpcodesniffer-composer-installer": "^0.4.1 || ^0.5 || ^0.6.2 || ^0.7",
 		"sirbrillig/phpcs-variable-analysis": "^2.11.1",
-		"squizlabs/php_codesniffer": "^3.5.5",
-		"wp-coding-standards/wpcs": "^2.3"
+		"squizlabs/php_codesniffer": "^3.7.1",
+		"wp-coding-standards/wpcs": "dev-develop"
 	},
 	"require-dev": {
-		"php-parallel-lint/php-parallel-lint": "^1.3.2",
 		"php-parallel-lint/php-console-highlighter": "^1.0.0",
+		"php-parallel-lint/php-parallel-lint": "^1.3.2",
 		"phpcompatibility/php-compatibility": "^9",
 		"phpcsstandards/phpcsdevtools": "^1.0",
 		"phpunit/phpunit": "^4 || ^5 || ^6 || ^7"
 	},
+	"minimum-stability": "dev",
+	"prefer-stable": true,
 	"config": {
 		"allow-plugins": {
 			"dealerdirect/phpcodesniffer-composer-installer": true
 		}
 	},
 	"scripts": {
-		"install-codestandards": "Dealerdirect\\Composer\\Plugin\\Installers\\PHPCodeSniffer\\Plugin::run",
-		"ruleset": "bin/ruleset-tests",
-		"lint": [
-			"bin/php-lint",
-			"bin/xml-lint"
+		"check-all": [
+			"@lint",
+			"@check-cs",
+			"@run-tests",
+			"@check-complete-strict",
+			"@ruleset"
 		],
-		"phpcs": "bin/phpcs",
-		"phpunit": "bin/unit-tests",
-		"coverage": "bin/unit-tests-coverage",
 		"check-complete": [
 			"@php ./vendor/phpcsstandards/phpcsdevtools/bin/phpcs-check-feature-completeness -q ./WordPressVIPMinimum"
 		],
-		"test": [
-			"@lint",
-			"@ruleset",
-			"@phpunit",
-			"@phpcs",
-			"@check-complete"
-		]
+		"check-complete-strict": [
+			"@php ./vendor/phpcsstandards/phpcsdevtools/bin/phpcs-check-feature-completeness ./WordPressVIPMinimum"
+		],
+		"check-cs": "bin/phpcs",
+		"lint": [
+			"bin/php-lint",
+			"bin/xml-lint"
+		],
+		"ruleset": "bin/ruleset-tests",
+		"run-tests": "bin/unit-tests",
+		"run-tests-coverage": "bin/unit-tests-coverage"
 	},
-	"support": {
-		"issues": "https://github.com/Automattic/VIP-Coding-Standards/issues",
-		"wiki": "https://github.com/Automattic/VIP-Coding-Standards/wiki",
-		"source": "https://github.com/Automattic/VIP-Coding-Standards"
+	"scripts-descriptions": {
+		"check-all": "Run all checks (lint, phpcs, feature completeness) and tests.",
+		"check-complete": "Check if all the sniffs have tests.",
+		"check-complete-strict": "Check if all the sniffs have unit tests and XML documentation.",
+		"check-cs": "Run the PHPCS script against the entire codebase.",
+		"lint": "Lint PHP files against parse errors.",
```

```suggestion
		"lint": "Check PHP files against parse errors.",
```

"Checking against parse errors" = "linting"

Also, looking at the script, the `lint` command also runs `xmllint`, should that be mentioned ?